pavement.py

```diff
@@ -73,9 +73,22 @@ def make_venv():
     bootstrap_venv(__here__ / path('env'), 'sideboard')
     develop_sideboard()
 
+def install_pip_requirements_in_dir(dir_of_requirements_txt):
+    path_to_pip = __here__ / path('env/bin/pip3')
```

Comment:
This line will break in Python 2.7 because it explicitly refers to ``pip3``.  I know that MAGFest doesn't care about older versions of Python, but most of the Sideboard projects where I work are still on 2.7, and Sideboard is explicitly a single-source project which works on either Python 2.7 or 3.x, so I'd like to keep it that way if possible.

I have at least 3 meetings this morning, but after those are done I can try checking out this branch on my dev box and see what we could put here that would work on both Python versions.